program_info/CMakeLists.txt

```diff
@@ -7,13 +7,14 @@ set(Launcher_DisplayName "${Launcher_CommonName}" PARENT_SCOPE)
 set(Launcher_UserAgent "${Launcher_CommonName}/${Launcher_RELEASE_VERSION_NAME}" PARENT_SCOPE)
 set(Launcher_ConfigFile "polymc.cfg" PARENT_SCOPE)
 set(Launcher_Git "https://github.com/PolyMC/PolyMC" PARENT_SCOPE)
+set(Launcher_DesktopFileName "org.polymc.PolyMC.desktop" PARENT_SCOPE)
```

Comment:
why would you set this but not use it in the Launcher_DESKTOP thing

Reply:
because Launcher_DESKTOP is the location of the file while building which is program_info/org.polymc.PolyMC.desktop and desktopfile is the location without any paths

Reply:
what, you can do `program_info/${Launcher_DESKTOPFILENAME}`, and configure the .in into Launcher_DESKTOPFILENAME

Reply:
ok i'll do that later